### PR TITLE
[Doc] update colocate join

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [x] 3.0
  - [x] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

**Review by amber-create — DISMISSED** [review]

File: docs/using_starrocks/Colocate_join.md [hunk 1]
```diff
@@ -54,6 +57,8 @@ If the specified Group does not exist, StarRocks automatically creates a Group t
 
 The Group belongs to a Database, and the name of the Group is unique within the Database. The full name of the Group is dbId_groupName in the internal storage, but the user only needs the groupName.
 
+The Group belongs to a Database. Even if you specify the same `colocate_with` property when creating tables in different Databases to ensure that these tables are colocated, each Database still contains a separate Group, which can be confirmed by using `show proc "/colocation_group"`. The Group name is unique within a Database. In the internal storage, the full name of the Group is `dbId_groupName`, but you only perceive `groupName`.
```
Comment (evelynzhaojie) [inline docs/using_starrocks/Colocate_join.md:60]: 1. The Group belongs to a Database. 这句话和58行的句子重复了。58 行的内容应该是要删除的。2. Even if you specify the same `colocate_with` property when creating tables in different Databases to ensure that these tables are colocated, each Database still contains a separate Group  这句话不好理解，中文也一样。我觉得就是想强调 group 是属于数据库层级的，如果不好改是不是可以删了。3. which can be confirmed by 表达建议改改，从show proc的回显字段里感觉confirm不出来这个信息呢，可以看看show proc那个命令。You can quey the information of Colocate Groups by using `show proc "/colocation_group"`.

File: docs/using_starrocks/Colocate_join.md [hunk 2]
```diff
@@ -38,6 +38,9 @@ The consistent data distribution and mapping guarantee that the data rows with t
 ### Table creation
 
 When creating a table, you can specify the attribute `"colocate_with" = "group_name"` in PROPERTIES to indicate that the table is a Colocate Join table and belongs to a specified Colocation Group.
+> **NOTE**
+>
+> From version 2.5.4, Colocate Join can be performed on tables from different databases. You only need to specify the same `colocate_with` property when tables are created.
```
Comment (evelynzhaojie) [inline docs/using_starrocks/Colocate_join.md:43]: when you create a table

File: docs/using_starrocks/Colocate_join.md [hunk 3]
```diff
@@ -52,7 +55,10 @@ PROPERTIES(
 
 If the specified Group does not exist, StarRocks automatically creates a Group that only contains the current table. If the Group exists, StarRocks checks to see if the current table meets the Colocation Group Schema. If so, it creates the table and adds it to the Group. At the same time, the table creates a partition and a tablet based on the data distribution rules of the existing Group.
 
-The Group belongs to a Database, and the name of the Group is unique within the Database. The full name of the Group is dbId_groupName in the internal storage, but the user only needs the groupName.
+A colocation group belongs to a database. The name of a colocation group is unique within a database. In the internal storage, the full name of the colocation group is `dbId_groupName`, but you only perceive `groupName`.
+> **NOTE**
+>
+> Even if you specify the same `colocate_with` property when creating tables in different databases to ensure that these tables are colocated, each database still contains a separate colocation group. You can query the information of colocation groups by using `show proc "/colocation_group"`.
```
Comment (evelynzhaojie) [inline docs/using_starrocks/Colocate_join.md:61]: 建议：Even though you specify the same Colocate Group to associate tables from different databases for Colocate Join, the Colocate Group exists in each of these databases. You can run xxx on each database to query the Colocate Group.

**Review by evelynzhaojie — APPROVED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Comment by hellolilyliuyi** [comment]
@mergify backport branch-3.0 branch-2.5

**Comment by hellolilyliuyi** [comment]
@mergify backport branch-3.0

**Pull request merged**